src/main/java/com/piekoszek/nowaksharedrent/controller/AuthController.java

```diff
@@ -0,0 +1,32 @@
+package com.piekoszek.nowaksharedrent.controller;
+
+import com.piekoszek.nowaksharedrent.model.Account;
+import com.piekoszek.nowaksharedrent.service.AuthService;
+import org.springframework.http.HttpStatus;
+import org.springframework.http.ResponseEntity;
+import org.springframework.web.bind.annotation.RequestBody;
+import org.springframework.web.bind.annotation.RequestMapping;
+import org.springframework.web.bind.annotation.RequestMethod;
+import org.springframework.web.bind.annotation.RestController;
+
+@RestController
+
+public class AuthController {
+    private AuthService authService;
+
+    public AuthController (AuthService authService) {
+        this.authService = authService;
+    }
+
+    @RequestMapping(value = "/auth/signup", method = RequestMethod.POST)
+    public ResponseEntity<Object> createAccount(@RequestBody Account account) {
```

Comment:
It should respond with JSON object like in task description. 
If you return String, like here, it will respond with raw text.
You need to wrap it in some object, for example SignupResponse/Response/MessageResponse or however you want to call it.
![image](https://user-images.githubusercontent.com/10707837/52482904-ccb38180-2bb2-11e9-8c08-cfe8864b45b2.png)